services/QuillLMS/app/models/concerns/translatable.rb

```diff
@@ -65,7 +65,12 @@ def create_translation_mappings
 
   def create_translation_mappings_with_text(translatable_text:, field_name: default_translatable_field)
     return unless translatable_text.is_a?(String) && translatable_text.present?
-    return unless translation_mappings.where(field_name:).empty?
+    return unless translation_mappings.joins(:english_text)
```

Comment:
Let's remove this double negative here (`unless finder_code.empty?`, and do `return if finder_code.exists?`